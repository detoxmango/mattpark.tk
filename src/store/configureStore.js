import { configureStore, getDefaultMiddleware } from '@reduxjs/toolkit'
import rootReducer from './reducers'
import promise from 'redux-promise'
import logger from 'redux-logger'

import { getIPv4, getIPv6, getGeoData } from './actions'

const configureAppStore = (preloadedState={}) => {
  const store = configureStore({
    reducer: rootReducer,
    middleware: [...getDefaultMiddleware(), logger, promise],
    preloadedState,
  })
  if (process.env.NODE_ENV !== 'production' && module.hot) {
    module.hot.accept('./reducers', () => (
      store.replaceReducer(rootReducer))
    )
  }
  // if (window !== undefined) {
  // store.dispatch(getIPv4())
  // store.dispatch(getIPv6())
  // store.dispatch(getGeoData())
  // }
  return store
}
export default configureAppStore
