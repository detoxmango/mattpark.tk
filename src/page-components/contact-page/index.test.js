import React from 'react'
import renderer from 'react-test-renderer'
import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'
import { MuiThemeProvider } from '@material-ui/core'

import getBaseTheme from '../../themes/base'
import ContactPage from './index'

const mockStore = configureStore([])

describe("ContactPage", () => {
  let store
  let tree

  beforeEach(() => {
    store = mockStore({
      app: {
        themeType: 'light',
        ipv4: '',
        ipv6: '',
        geoData: '',
      }
    })

    tree = renderer.create(
      <Provider store={store}>
        <MuiThemeProvider theme={getBaseTheme(store.getState().app.themeType)}>
          <ContactPage />
        </MuiThemeProvider>
      </Provider>
    ).toJSON()
  })
  // it("renders correctly", () => {
  //   expect(tree).toMatchSnapshot()
  // })
  it("passes", () => {
    expect(true).toBe(true)
  })
})
