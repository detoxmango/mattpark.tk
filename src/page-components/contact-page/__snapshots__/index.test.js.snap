// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactPage renders correctly 1`] = `
<div
  className="Component-root-1"
>
  <div
    className="Component-root-2"
  >
    <h1
      className="MuiTypography-root Component-myName-3 MuiTypography-h1"
    >
      Matt Park
    </h1>
    <div
      className="Component-myInfo-4"
    >
      <h2
        className="MuiTypography-root MuiTypography-h2"
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineNone Component-phoneLink-5 MuiTypography-colorPrimary"
          href="tel:+1201-591-3323"
          onBlur={[Function]}
          onFocus={[Function]}
          target="_top"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15 12h2c0-2.76-2.24-5-5-5v2c1.66 0 3 1.34 3 3zm4 0h2c0-4.97-4.03-9-9-9v2c3.87 0 7 3.13 7 7zm1 3.5c-1.25 0-2.45-.2-3.57-.57-.1-.03-.21-.05-.31-.05-.26 0-.51.1-.71.29l-2.2 2.2c-2.83-1.44-5.15-3.75-6.59-6.59l2.2-2.21c.28-.26.36-.65.25-1C8.7 6.45 8.5 5.25 8.5 4c0-.55-.45-1-1-1H4c-.55 0-1 .45-1 1 0 9.39 7.61 17 17 17 .55 0 1-.45 1-1v-3.5c0-.55-.45-1-1-1zM5.03 5h1.5c.07.88.22 1.75.45 2.58l-1.2 1.21c-.4-1.21-.66-2.47-.75-3.79zM19 18.97c-1.32-.09-2.6-.35-3.8-.76l1.2-1.2c.85.24 1.72.39 2.6.45v1.51z"
            />
          </svg>
           
          <span
            className="Component-phoneText-6"
          >
            (201) 591 - 3323
          </span>
        </a>
        <br />
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineNone Component-emailLink-7 MuiTypography-colorPrimary"
          href="mailto:mattparksolutions@gmail.com"
          onBlur={[Function]}
          onFocus={[Function]}
          target="_top"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V8l8 5 8-5v10zm-8-7L4 6h16l-8 5z"
            />
          </svg>
           
          <span
            className="Component-emailText-8"
          >
            mattparksolutions@gmail.com
          </span>
        </a>
      </h2>
      <h2
        className="MuiTypography-root Component-socialMediaLinks-9 MuiTypography-h2"
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
          href="https://github.com/mattparksolutions"
          onBlur={[Function]}
          onFocus={[Function]}
          rel="noopener noreferrer"
          target="_blank"
          title="GitHub"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
            />
          </svg>
        </a>
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
          href="https://linkedin.com/in/mattparksolutions"
          onBlur={[Function]}
          onFocus={[Function]}
          rel="noopener noreferrer"
          target="_blank"
          title="LinkedIn"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
            />
          </svg>
        </a>
      </h2>
    </div>
  </div>
  <form
    className="Component-form-55"
    onSubmit={[Function]}
  >
    <div
      className="MuiFormControl-root MuiTextField-root"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root Component-label-56 MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
        data-shrink={false}
      >
        Name
        <span
          aria-hidden={true}
          className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
        >
           
          *
        </span>
      </label>
      <div
        className="MuiInputBase-root MuiOutlinedInput-root Component-outlinedInput-58 MuiInputBase-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          aria-label="name"
          autoFocus={false}
          className="MuiInputBase-input MuiOutlinedInput-input"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="text"
          value=""
        />
        <fieldset
          aria-hidden={true}
          className="PrivateNotchedOutline-root-118 MuiOutlinedInput-notchedOutline Component-notchedOutline-59"
        >
          <legend
            className="PrivateNotchedOutline-legendLabelled-120"
          >
            <span>
              Name
               *
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
    <div
      className="MuiFormControl-root MuiTextField-root"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root Component-label-56 MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
        data-shrink={false}
      >
        Email
        <span
          aria-hidden={true}
          className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
        >
           
          *
        </span>
      </label>
      <div
        className="MuiInputBase-root MuiOutlinedInput-root Component-outlinedInput-58 MuiInputBase-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          aria-label="email"
          autoFocus={false}
          className="MuiInputBase-input MuiOutlinedInput-input"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="text"
          value=""
        />
        <fieldset
          aria-hidden={true}
          className="PrivateNotchedOutline-root-118 MuiOutlinedInput-notchedOutline Component-notchedOutline-59"
        >
          <legend
            className="PrivateNotchedOutline-legendLabelled-120"
          >
            <span>
              Email
               *
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
    <div
      className="MuiFormControl-root MuiTextField-root"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root Component-label-56 MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined Mui-required Mui-required"
        data-shrink={false}
      >
        Message
        <span
          aria-hidden={true}
          className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
        >
           
          *
        </span>
      </label>
      <div
        className="MuiInputBase-root MuiOutlinedInput-root Component-outlinedInput-58 MuiInputBase-formControl MuiInputBase-multiline MuiOutlinedInput-multiline"
        onClick={[Function]}
      >
        <textarea
          aria-invalid={false}
          aria-label="message"
          autoFocus={false}
          className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          rows={8}
          value=""
        />
        <fieldset
          aria-hidden={true}
          className="PrivateNotchedOutline-root-118 MuiOutlinedInput-notchedOutline Component-notchedOutline-59"
        >
          <legend
            className="PrivateNotchedOutline-legendLabelled-120"
          >
            <span>
              Message
               *
            </span>
          </legend>
        </fieldset>
      </div>
    </div>
    <label
      className="MuiFormControlLabel-root"
    >
      <span
        aria-disabled={false}
        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-134 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={null}
      >
        <span
          className="MuiIconButton-label"
        >
          <input
            aria-label="privacy policy"
            checked={false}
            className="PrivateSwitchBase-input-137"
            data-indeterminate={false}
            onChange={[Function]}
            required={true}
            type="checkbox"
          />
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
            />
          </svg>
        </span>
      </span>
      <span
        className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
      >
        I understand that Matt will securely hold my data in accordance with their privacy policy.
      </span>
    </label>
    <button
      aria-label="send"
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSizeLarge MuiButton-sizeLarge"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="submit"
    >
      <span
        className="MuiButton-label"
      >
        Send
        <span
          className="MuiButton-endIcon MuiButton-iconSizeLarge"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M2.01 21L23 12 2.01 3 2 10l15 2-15 2z"
            />
          </svg>
        </span>
      </span>
    </button>
  </form>
</div>
`;
